packages-native/rating/package.json

```diff
@@ -1,31 +1,32 @@
 {
-    "name": "rating",
-    "widgetName": "Rating",
-    "version": "0.1.5",
-    "license": "Apache-2.0",
-    "repository": {
-        "type": "git",
-        "url": "https://github.com/mendix/widgets-resources.git"
-    },
-    "config": {
-        "widgetPath": "../test-project/mxproject/widgets",
-        "projectPath": "../test-project/mxproject"
-    },
-    "packagePath": "com.mendix.widget.native",
-    "scripts": {
-        "start": "..\"/../node_modules/.bin/pluggable-widgets-tools\" start:ts:native -- --subprojectPath packages-native/rating",
-        "build": "..\"/../node_modules/.bin/pluggable-widgets-tools\" build:ts:native --subprojectPath packages-native/rating",
-        "release": "..\"/../node_modules/.bin/pluggable-widgets-tools\" release:ts:native --subprojectPath packages-native/rating",
-        "lint": "..\"/../node_modules/.bin/eslint\" --config ../../.eslintrc.js --ext .jsx,.js,.ts,.tsx src/",
-        "version": "npm run release",
-        "postversion": "..\"/../node_modules/.bin/ts-node\" ../../scripts/release/UpdateGithubRelease.ts --project ../../scripts/tsconfig.json"
-    },
-    "dependencies": {
-        "@native-mobile-resources/util-widgets": "^1.0.0",
-        "@widgets-resources/piw-utils": "^1.0.0",
-        "react-native-star-rating": "^1.1.0"
-    },
-    "devDependencies": {
-        "@types/react-native-star-rating": "^1.0.1"
-    }
+  "name": "rating",
+  "widgetName": "Rating",
+  "version": "0.1.5",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mendix/widgets-resources.git"
+  },
+  "config": {
+    "widgetPath": "../test-project/mxproject/widgets",
+    "projectPath": "../test-project/mxproject"
+  },
+  "packagePath": "com.mendix.widget.native",
+  "scripts": {
+    "start": "pluggable-widgets-tools start:ts:native -- --subprojectPath packages-native/rating",
+    "build": "pluggable-widgets-tools build:ts:native --subprojectPath packages-native/rating",
+    "release": "pluggable-widgets-tools release:ts:native --subprojectPath packages-native/rating",
+    "lint": "..\"/../node_modules/.bin/eslint\" --config ../../.eslintrc.js --ext .jsx,.js,.ts,.tsx src/",
+    "version": "npm run release",
```

Comment:
Why are we keeping duplicate scripts ? this will be the equiv of `npm run release` anyways